kubernetes-manifests/userservice.yaml

```
@@ -80,7 +80,7 @@ spec:
           timeoutSeconds: 10
         resources:
           requests:
-            cpu: 200m
+            cpu: 500m
```

As Olivier mentioned, the `kubernetes-manifests/userservice.yaml` should not be manually updated:
```suggestion
            cpu: 200m
```
Updates to it happen automatically when we create a Bank of Anthos release.
The contents of https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/main/dev-kubernetes-manifests are copied over.
This means that https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/main/dev-kubernetes-manifests is the file that we should make manual updates to.

**Suggestion:** It would be cool if each file inside `kubernetes-manifests/` mentioned this as a warning (inside a comment).